Sources/AWSLambdaRuntimeCore/LambdaRuntimeClient.swift

```diff
@@ -13,13 +13,8 @@
 //===----------------------------------------------------------------------===//
 
 import Logging
-#if compiler(>=5.6)
-@preconcurrency import NIOCore
-@preconcurrency import NIOHTTP1
-#else
```

Comment:
Same as before, Xcode 14 says these aren't needed. Will revert this change if it breaks something before Swift 5.7